src/latexify/codegen/function_codegen.py

```diff
@@ -517,6 +517,16 @@ def visit_If(self, node: ast.If) -> str:
         latex += self.visit(node)
         return latex + r", & \mathrm{otherwise} \end{array} \right."
 
+    def visit_IfExp(self, node: ast.IfExp) -> Any:
+        """Visit an ifexp node"""
+        latex = r"\left\{ \begin{array}{ll} "
+        cond_latex = self.visit(node.test)
+        true_latex = self.visit(node.body)
+        false_latex = self.visit(node.orelse)
```

Comment:
Note: I can make some follow-up pull requests for this comment.

I think this line does not process the recursion. e.g.:

```python
if a:
    return x
elif b:
    return y
else:
    return z
```

and

```python
return x if a else (y if b else z)
```

should express the same logic, but this impl may generates different result.

Reply:
<img width="878" alt="Screenshot 2022-11-15 at 21 24 42" src="https://user-images.githubusercontent.com/48125086/202028349-c0972c75-0d35-4432-a3a2-21e2ebad61bb.png">

I was able to process the recursion you highlighted, but wanted to know if we need to support recursion within both code blocks of the `if  then  else` statement like shown in the 2nd example.

Reply:
In `visit_If` the current implementation processes only `orelse` subtree. I think we could choose the same behavior at this point.